packages/mui-joy/src/styles/styleFunctionSx.test.js

```diff
@@ -0,0 +1,112 @@
+import { expect } from 'chai';
+import styleFunctionSx from './styleFunctionSx';
+
+describe('styleFunctionSx', () => {
```

Comment:
Would it be worth adding tests for some breakpoints too?

Reply:
added!